src/abstractions/src/promptTemplate/KernelPromptTemplate.ts

```diff
@@ -0,0 +1,13 @@
+import { PromptTemplate } from './PromptTemplate';
+import { handlebarsPromptTemplate } from './handlebarsPromptTemplate';
+
+export class KernelPromptTemplate implements PromptTemplate {
+  constructor(
+    private readonly template: string,
+    private readonly args: object
```

Comment:
I think this type needs to be changed to `KernelArguments`?

Reply:
Updated in commit https://github.com/afshinm/semantic-kernel-js/commit/1a447c40b3ff2a491874434223830051508fda4b